rif/relay/architecture.md

```diff
@@ -88,28 +100,44 @@ When a Relay Manager unauthorised a Relay Hub, it means it is unstaking from it,
 
 Unstaking has a predefined delay (in blocks). This is intended to prevent the Relay Manager from unstaking before a slashing that was going to occur.
 
-### Relay & Deploy Verifier
-Abstract contracts that authorizes a specific relay or deploy request (see the [Relay & Deploy Requests](#relay--deploy-requests) section).
+### Relay Worker
+An EOA that belongs to only one Relay Manager. It’s the sender of the Relay Request. Since the Relay Worker is the sender of the request is the one that pays for the fee and gas consumption from each transaction. It also collects the fees in ERC20 that are charged for relaying the transactions. 
```

Comment:
```suggestion
An EOA that belongs to only one Relay Manager. Since the Relay Worker is the sender of the request, it is the one that pays for the gas fees for each transaction. It **may** also collect the fees in ERC20 that are charged for relaying the transactions. 
```